src/app.js

```diff
@@ -0,0 +1,35 @@
+import './main.css';
+
+const taskList = document.getElementById('task-list');
+
+const tasks = [
+  {
+    description: 'wash the dishes',
+    completed: 'false',
+    index: '0',
+  },
+  {
+    description: 'complete To Do list project',
+    completed: 'true',
+    index: '1',
+  },
+];
+
+const createCheckbox = () => {
+  const taskCheckbox = document.createElement('input');
+  taskCheckbox.type = 'checkbox';
+  return taskCheckbox;
+};
+
+for (let index = 0; index < tasks.length; index += 1) {
+  const listItem = document.createElement('li');
+  listItem.appendChild(createCheckbox());
+  listItem.appendChild(document.createTextNode(tasks[index].description));
+  taskList.appendChild(listItem);
+}
+
+// Create a horizontal line element
+const hr = document.querySelector('.hr');
+hr.innerHTML = '<hr>';
+const clear = document.getElementById('clear');
+clear.innerHTML = 'Clear all completed';
```

Comment:
- [ ] Based on the project requirements, you are expected to create a function to iterate over the tasks array and populate an HTML list item element for each task. See example below 👇🏾 

**_Note_** - The index should begin from `1` and not `0`. 


```javascript
const tasks = [
        { index: 1, description: "Task 1" },
        { index: 2, description: "Task 2" },
        { index: 3, description: "Task 3" },
      ];

      function renderTasks() {
        const taskList = document.getElementById("task-list");

        tasks.sort((a, b) => a.index - b.index); // Optional, but its better to have this

        tasks.forEach((task) => {
          const li = document.createElement("li");
          li.textContent = task.description;
          taskList.appendChild(li);
        });
      }    
```

Kindly use a function to iterate and display the tasks. 

- [ ] Also, it is expected that on page load, you render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task. You should use Windows `load` events which fire when the **_whole page_** has loaded, including all dependent resources such as stylesheets, scripts, iframes, and images.

```javascript
  window.addEventListener("load", renderTasks);
```